source/Camera/Basic/CaptureTutorial.md

```diff
@@ -0,0 +1,214 @@
+## Introduction
+
+This tutorial describes how to use Zivid SDK to capture point clouds and 2D images.
+
+1. [Initialize](#initialize)
+2. [Connect](#connect)
+   1. [Specific Camera](#connect---specific-camera)
+   2. [Virtual Camera](#connect---virtual-camera)
+3. [Configure](#configure)
+   1. [Capture Assistant](#capture-assistant)
+   2. [Manual Configuration](#manual-configuration)
+      1. [Single](#single-frame)
+      2. [HDR](#hdr-frame)
+      3. [From File](#from-file)
+      4. [2D](#2d-settings)
+4. [Capture](#capture)
+    1. [HDR](#capture-hdr)
+    2. [2D](#capture-2d)
+5. [Save](#save)
```

Comment:
Can we add:

5. [Save](#save)
    1. [2D](#save-2d)

Like we did in python: https://github.com/zivid/python-samples/commit/4406342f5eae91e649fa393a8bb1bc9f253ae598